Makefile

```diff
@@ -153,6 +153,10 @@ test-image-all:
 .PHONY: test-e2e
 test-e2e: test-e2e-1.26
 
+.PHONY: test-e2e-local
```

Comment:
Don't understand idea of `local`. If you want to implement cleaning images why not add removing docker images in `clean` makefile target?

Reply:
Makes sense, patched in https://github.com/kubernetes-sigs/metrics-server/pull/1183/commits/31ee3002d93e43aefe744ea02682c5378c93effb.